cypress/features/regression/experimental/decimalInputClassic.feature

```diff
@@ -0,0 +1,136 @@
+Feature: Decimal classic component
+  I want to change Decimal classic component properties
+
+  Background: Open Decimal component classic page
+    Given I open "Experimental-Decimal-Input" component page classic
+
+  @positive
+  Scenario Outline: Change Decimal component fieldHelp to <fieldHelp>
+    When I set fieldHelp to "<fieldHelp>"
+    Then fieldHelp on preview is set to "<fieldHelp>"
+    Examples:
+      | fieldHelp                |
+      | Sample text              |
+      | 1234567890               |
+      | áéíóú¿¡üñ                |
+      | !@#$%^*()_+-=~[];:.,?{}  |
+      | ÄÖÜßäöüß                 |
+      | <>                       |
+
+  @positive
+  Scenario Outline: Change Decimal component label to <label>
+    When I set label to "<label>"
+    Then label on preview is "<label>"
+    Examples:
+      | label                    |
+      | Sample text              |
+      | 1234567890               |
+      | áéíóú¿¡üñ                |
+      | !@#$%^*()_+-=~[];:.,?{}  |
+      | ÄÖÜßäöüß                 |
+      | <>                       |
+
+  @positive
+  Scenario Outline: Change Decimal input align to the <direction>
+    When I select align to "<direction>"
+    Then input direction is "<direction>"
+    Examples:
+      | direction |
+      | left      |
+      | right     |
+
+  @positive
+  Scenario Outline: Change Decimal component label help to <label>
+    When I set label to "<label>"
```

Comment:
Maybe we should use 'Given' instead of 'When' ? I think that 'I hover mouse onto help icon' step is the proper action step before a result

Reply:
fixed